test/e2e/pod_stop_test.go

```diff
@@ -137,6 +137,5 @@ var _ = Describe("Podman pod stop", func() {
 		session = podmanTest.Podman([]string{"pod", "stop", podid1, "doesnotexist"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(125))
-		Expect(podmanTest.NumberOfContainersRunning()).To(Equal(0))
```

Comment:
should this still hold?

Reply:
the behaviour has changed slightly

Reply:
How so? We should be killing everything, including the infra container

Reply:
the way the error handling is being done now wrt to BOGUS pod names is if the input has bogus pod identifiers in it, we will return an error first.  it used to be we tried a brute force regardless.